pkg/workloads/cortex/tf_api/api.py

```diff
@@ -205,7 +219,10 @@ def predict(deployment_name, api_name):
     response = {}
 
     if not util.is_dict(payload) or "samples" not in payload:
-        return prediction_failed('top level "samples" key not found in request')
+        message = 'top level "samples" key not found in request'
+        if debug:
+            message += "(payload: {})".format(util.pp_str_flat(payload))
```

Comment:
the trailing `)` might be confusing to users. Maybe `+= " | payload: {}"` or `+= "; payload: {}"` or something similar